cytoscape/cytoscape-web
# Example apps enabled

This branch has three example apps in the settings files. If this works, you can see those three apps in the Apps --> Settings.

## How to test this branch

1. Clone this repository: https://github.com/cytoscape/cytoscape-web-app-examples
2. Make sure you are in the development branch. 
3. cd to the directory and npm install && npm run dev. This command starts multiple apps in your local machine.
4. In different terminal, run Cytoscape Web local instance
5. Open localhost:5500
6. If you can see some menu items under Apps, external apps are successfully loaded. 

Note that the settings are configured to run local instances and it needs updates when you want to test those in remote servers.
